.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/foundry"]
 	path = lib/foundry
-	url = git@github.com:jxom/foundry.git
+	url = git@github.com:wevm/foundry.git
+	branch = jxom/eip-3074
```

Comment:
should this be rebased on #23 ?

Reply:
update: either submodule configuration works, so this change should be fine